simple/stats/config.md

```diff
@@ -36,7 +36,16 @@ The config parameters for the files to be imported should be specified in a `con
         "Provenance2 Name": "http://source1.com/provenance2"
       }
     }
-  }
+  },
+  "database": {
+    "type": "cloudMySql",
+    "params": {
+        "instance": "datcom-website-dev:us-central1:dc-graph",
```

Comment:
wonder if we should write the testing data to live db? Also should remove the user and password here?

Reply:
We are not. This is only in the doc. The actual sample continues to write to sqlite. The password is fake.

Reply:
I see. Then maybe use something like
`instance: "<project:region:instance>"`
`user:<YOUR_USER_NAME>`...

Also, is  the db name "datacommons" dynamic or fixed?

Reply:
Dynamic.